File: other/Fibonacci.java
public class Fibonacci {

    private static void fibo(int num){

        int m = 0;
        int n = 1;

        for (int i = 0; i < num; i++){

            System.out.println(m);
            int sum = m + n;
            m = n;
            n = sum;
        }
    }

    public static void main(String[] args){

        int num = 6;
        fibo(num);
    }
}

File: other/Fibonacci.js

const fibonacci = (num) => {

    let a = 0;
    let b = 1;

    for (let i = 0; i < num; i++){

        console.log(a);
        let sum = a + b;
        a = b;
        b = sum;
    }
}

const num = 7;
console.log(fibonacci(num));